src/templates/_layouts/parts-kit.html

```diff
@@ -1,6 +1,12 @@
 {% extends craft.viget.partsKit.getConfig('layout') %}
 
 {% block content %}
+    <style>
+        .parts-kit {
+            {{ craft.viget.partsKit.getTheme() | join() }}
```

Comment:
Also wondering if it makes sense to move the join into php. Any reason to keep it here?